bare-metal/dedibox/how-to/activate-ipv6-block.mdx

```diff
@@ -16,7 +16,7 @@ dates:
 <Macro id="requirements" />
 
 <Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```

Comment:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

Reply:
I was going to suggest edits to this file, but I've just updated the entire missing category on this PR: https://github.com/scaleway/docs-content/pull/3011

If you want, you can just remove them here, so there are no conflicts. As you prefer.